### PR TITLE
Typos a/an

### PR DESCRIPTION
**Pull request by dilyanpalauzov**

### CODE REVIEW

File: bundles/org.openhab.binding.webthing/src/main/java/org/openhab/binding/webthing/internal/discovery/WebthingDiscoveryService.java [hunk 1]
```diff
@@ -229,7 +229,7 @@ private Set<DiscoveryResult> discoverWebThing(ServiceInfo serviceInfo) {
             // endpoints supporting multiple WebThings.
             //
             // In the routine below the enpoint will be checked for single WebThings first, than for multiple
-            // WebThings if a ingle WebTHing has not been found.
+            // WebThings if an ingle WebTHing has not been found.
```
Comment (jlaur) [inline bundles/org.openhab.binding.webthing/src/main/java/org/openhab/binding/webthing/internal/discovery/WebthingDiscoveryService.java:232]:
```suggestion
            // WebThings if a single WebThing has not been found.
```

**Review by jlaur — CHANGES_REQUESTED** [review]
Thank you! While reviewing I found a few possible additional mistakes in the changed lines, can you check them also?

File: bundles/org.openhab.binding.webthing/src/main/java/org/openhab/binding/webthing/internal/discovery/WebthingDiscoveryService.java [hunk 2]
```diff
@@ -229,7 +229,7 @@ private Set<DiscoveryResult> discoverWebThing(ServiceInfo serviceInfo) {
             // endpoints supporting multiple WebThings.
             //
             // In the routine below the enpoint will be checked for single WebThings first, than for multiple
-            // WebThings if a ingle WebTHing has not been found.
+            // WebThings if a single WebTHing has not been found.
```
Comment (jlaur) [inline bundles/org.openhab.binding.webthing/src/main/java/org/openhab/binding/webthing/internal/discovery/WebthingDiscoveryService.java:232]:
```suggestion
            // WebThings if a single WebThing has not been found.
```

**Review by jlaur — APPROVED** [review]
LGTM

**Comment by jlaur** [comment]
@dilyanpalauzov - there is one conflict to resolve.

**Review by wborn — APPROVED** [review]
Thanks!

**Review by druciak — COMMENTED** [review]
Approving changes for Satel binding.

**Pull request merged**